wampy/transports/websocket/frames.py

```diff
@@ -85,7 +85,14 @@ def __init__(self, bytes):
         self.body = bytes
 
     def __len__(self):
-        return len(self.body)
+        # UTF-8 is an unicode encoding which uses more than one byte for
+        # special characters. calculating the length needs consideration.
+        try:
+            unicode_body = self.body.decode("utf-8")
```

Comment:
i think this is okay. WAMP protocol dictates this encoding. possibly though i could enforce this at an earlier point in the serialisation process.